flink-kubernetes/src/main/java/org/apache/flink/kubernetes/KubernetesClusterDescriptor.java

```diff
@@ -110,10 +113,18 @@ public String getClusterDescription() {
                 // Flink client will always use Kubernetes service to contact with jobmanager. So we
                 // have a pre-configured web monitor address. Using StandaloneClientHAServices to
                 // create RestClusterClient is reasonable.
-                return new RestClusterClient<>(
-                        configuration,
-                        clusterId,
-                        new StandaloneClientHAServices(getWebMonitorAddress(configuration)));
+                String webMonitorAddress = getWebMonitorAddress(configuration);
```

Comment:
I would move this into the factory because we don't know whether `configuration` won't be changed.

Reply:
I made a wrong thought here, done!